blocks_vertical/event.js

```diff
@@ -77,13 +77,10 @@ Blockly.Blocks['event_whenbroadcastreceived'] = {
       "message0": "when I receive %1",
       "args0": [
         {
-          "type": "field_dropdown",
+          "type": "field_variable",
           "name": "BROADCAST_OPTION",
-          "options": [
-            ['message1', 'message1'],
-            ['message2', 'message2'],
-            ['new message', 'new message']
-          ]
+          "variableTypes": ["broadcast_msg"],
+          "variable": "message1"
```

Comment:
Is "message1" going to be the same for all languages?  Blockly usually uses messages here so that they can be translated.

For instance, the default name [here](https://developers.google.com/blockly/guides/create-custom-blocks/variables#untyped_variable_blocks) is given by "BKY_VARIABLES_DEFAULT_NAME"